okta/resource_okta_user_test.go

```diff
@@ -233,6 +234,17 @@ func TestAccOktaUser_updateAllAttributes(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "email", email),
 				),
 			},
+			{
+				Config: minimalConfigWithCredentials,
```

Comment:
Once we add the `ForceNew` this will actually recreate this resource and test the create logic.